core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -388,129 +379,87 @@ private DefaultFunctionResolver addtime() {
             nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime), TIME, TIME, TIME),
         implWithProperties(
             nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime), TIME, TIME, DATE),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime), TIME, TIME, DATETIME),
         implWithProperties(
             nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
             TIME,
             TIME,
             TIMESTAMP),
         implWithProperties(
             nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
-            DATETIME,
-            DATETIME,
+            TIMESTAMP,
+            DATE,
             TIME),
         implWithProperties(
             nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
-            DATETIME,
-            DATETIME,
-            DATE),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
-            DATETIME,
-            DATETIME,
-            DATETIME),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
-            DATETIME,
-            DATETIME,
-            TIMESTAMP),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime), DATETIME, DATE, TIME),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime), DATETIME, DATE, DATE),
-        implWithProperties(
-            nullMissingHandlingWithProperties(DateTimeFunction::exprAddTime),
-            DATETIME,
+            TIMESTAMP,
             DATE,
-            DATETIME),
+            DATE),
```

Comment:
Why was this replaced with `date` and not `timestamp`?

Reply:
Same reason as [above](https://github.com/Bit-Quill/opensearch-project-sql/pull/336/files/2431eb61aca74dcdb1537d0e694b95bcc4ce26ad#r1293969489)